### PR TITLE
Add methods for comparing Moments

### PR DESCRIPTION
**Pull request by jarekkozak**
Hi,
Reiplemented from original moment methods isBefore, isAfter, isSame and new one which is useful for me isBetween.
+3 helper methods toUTC, toArray and setDateFromArray (just realized maybe setMomentFromArray would be better).
Best regards,
Jarek

### CODE REVIEW

File: src/Moment.php [hunk 1]
```diff
@@ -990,4 +990,132 @@ private function formatOrdinal($number)
...
+     * @param string|Moment $minDateTime
+     * @param string|Moment $maxDateTime
+     * @param boolean $closed - true = include endpoints, false = do not
+     * @param string $period 'seconds|minute|hour|day|month|year'
+     * @return boolean
+     */
+    public function isBetween($minDateTime, $maxDateTime, $closed = true, $period = 'seconds'){
+        if($closed){
+            return (bool)(!$this->isBefore($minDateTime,$period) && !$this->isAfter($maxDateTime,$period));
+        }else{
```
Comment (fightbulc) [inline src/Moment.php]: you don't need the else condition. just push the return. its much cleaner.

File: src/Moment.php [hunk 2]
```diff
@@ -990,4 +990,132 @@ private function formatOrdinal($number)
...
+        }
+    }
+    
+    /**
+     * Returns Moment as array
+     * 
+     * @param bool $assoc - true association array
+     * @return array
+     */
+    public function toArray($assoc = true)
```
Comment (fightbulc) [inline src/Moment.php]: I think I rather add an import/export method which expects a closure or helper class for handling the process.

**Comment by fightbulc** [comment]
Jarek, I think I will pull and make an adjustment to the last two methods.

**Comment by jarekkozak** [comment]
Ok. up to you. Feel free:) 
Best regards,
Jarek

**Pull request merged**